<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edtCode">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="edtName">
     <property name="maxLength">
      <number>64</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>&amp;Наименование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QCheckBox" name="chkTabRegistry">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblTabRegistry">
     <property name="text">
      <string>Регистратура: Картотека</string>
     </property>
     <property name="buddy">
      <cstring>chkTabRegistry</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblTabEvents">
     <property name="text">
      <string>Регистартура: Обращения</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblAmbCard">
     <property name="toolTip">
      <string>используйте {Name}
для подстановки
значения параметра
Name</string>
     </property>
     <property name="text">
      <string>Регистратура: Медкарта</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblTabActions">
     <property name="toolTip">
      <string>используйте {Name}
для подстановки
значения параметра
Name</string>
     </property>
     <property name="text">
      <string>Регистратура: Обслуживание</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="chkTabActions">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="chkTabEvents">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="chkTabAmbCard">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblTemplate">
     <property name="text">
      <string>Шаблон URL</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="edtTemplate"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>Код</string>
     </property>
     <property name="visible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Вкладки, в которых подхватывается данное действие:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtName</tabstop>
  <tabstop>chkTabRegistry</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
